### PR TITLE
Fix issue while storing empty email

### PR DESCRIPTION
**Pull request by mohityadav766**
### Describe your changes:

Fixed the issue with empty Organization

I worked on ... because ...

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/Team/TeamDetails/TeamDetailsV1.tsx [hunk 1]
```diff
@@ -41,6 +43,7 @@ import {
   GlobalSettingsMenuCategory,
 } from 'constants/GlobalSettings.constants';
 import { DROPDOWN_ICON_SIZE_PROPS } from 'constants/ManageButton.constants';
+import { validEmailRegEx } from 'constants/regex.constants';
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Team/TeamDetails/TeamDetailsV1.tsx:46]:
```suggestion
import { EMAIL_REG_EX } from 'constants/regex.constants';
```

File: openmetadata-ui/src/main/resources/ui/src/components/Team/TeamDetails/TeamDetailsV1.tsx [hunk 2]
```diff
@@ -886,34 +889,48 @@ const TeamDetailsV1 = ({
...
-            size="small"
-            type="primary"
-            onClick={handleUpdateEmail}>
-            <CheckOutlined />
-          </Button>
-        </Space>
+        <Form
+          initialValues={{ email: currentTeam.email }}
+          onFinish={handleUpdateEmail}>
+          <Space align="baseline">
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Team/TeamDetails/TeamDetailsV1.tsx:895]: Not sure why we need Space for form layout

**Review by chirag-madlani — APPROVED** [review]
@Sachin-chaurasiya minor comment which we can address later

File: openmetadata-ui/src/main/resources/ui/src/components/Team/TeamDetails/TeamDetailsV1.tsx (hunk 2 above)
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/components/Team/TeamDetails/TeamDetailsV1.tsx:895]: @chirag-madlani , we have one input and 2 buttons so to align them we need space.

**Pull request merged**